### DISCUSSION TITLE
Light Bleed Compensation suggested settings/calibration

### DISCUSSION THREAD

**Discussion by futrtrubl** [sn4k3/UVtools#358] [Q&A]
Does anyone have suggested settings for the Light Bleed Compensation tool or a way to figure out such? I am noticing some overexposure on the plate facing surfaces of prints but not sure how to quantify and compensate for it.

**Comment by sn4k3** [comment]
I have no data on this. It require someone to test.
But you can start with default values, just run the tool without adjust settings and run the print. Then evaluate and iterate from there. 
The default settings are my guess to opaque resins, also it dim levels are safe to run
